code/modules/mob/living/carbon/xenomorph/castes/Crusher.dm

```diff
@@ -39,6 +39,12 @@
 	tier = 3
 	drag_delay = 6 //pulling a big dead xeno is hard
 
+/mob/living/carbon/xenomorph/crusher/recalculate_actions()
+	. = ..()
+	pull_multiplier *= 0.5
```

Comment:
Don't do this here. Just change the value in the define for /mob/living/carbon/xenomorph/crusher